src/ic-cdk/src/api/call.rs

```diff
@@ -212,33 +207,16 @@ pub fn reply<T: ArgumentEncoder>(reply: T) {
 pub mod funds {
     use super::ic0;
 
-    pub enum Unit {
-        Cycle,
-        IcpToken,
-    }
-
-    impl Unit {
-        pub fn to_bytes(&self) -> Vec<u8> {
-            match self {
-                Unit::Cycle => vec![0],
-                Unit::IcpToken => vec![1],
-            }
-        }
-    }
-
-    pub fn available(unit: Unit) -> i64 {
-        let bytes = unit.to_bytes();
-        unsafe { ic0::msg_funds_available(bytes.as_ptr() as i32, bytes.len() as i32) }
+    pub fn available() -> i64 {
```

Comment:
Nit: It might be worth naming this now `cycles_available` to make it clear what this number that it returns is. Previously, we had `available(unit)` at the call site so it was self-describing via the argument.

Reply:
As this is inside the `funds` module, I thoughts it's fine.  I do not quite understand the goal of the module though.  Maybe the module should be renamed to `cycles`.  I will let the repo owners respond on this.

Reply:
The module should probably be removed. It was when we had plans to have multiple types of tokens held by canisters. 

This library needs a lot of love right now. Lots of tech debt from even when I was building the wallet.

Reply:
Cool, updated.